recipes/statannot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "statannot" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4600c2fa1f682d493a4ad0c1c4501b32a3d8d3a55877ec97bcf4e44fbd160cb9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - numpy
+    - seaborn >=0.9.0
+    - matplotlib
+    - pandas
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - statannot
+
+about:
+  home: "https://github.com/webermarcolivier/statannot"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt 
```

Comment:
Should update to `LICENSE` and use the one in the tarball.

Reply:
Like this?

Reply:
Yup!